src/main/kotlin/com/atlassian/performance/tools/report/api/judge/ActionReport.kt

```diff
@@ -0,0 +1,16 @@
+package com.atlassian.performance.tools.report.api.judge
+
+import com.atlassian.performance.tools.jiraactions.api.ActionType
+import com.atlassian.performance.tools.report.api.junit.JUnitReport
+
+/**
+ * Action aware report
+ *
+ * @property action Action associated with this report.
+ * @property nonExceptional True if the failure in this report is expected sometimes as a part of a normal flow, false otherwise.
+ */
+interface ActionReport : JUnitReport {
+
+    val action: ActionType<*>
+    val nonExceptional: Boolean
```

Comment:
We've possibly got a triple negation here. If this is `false`, then the report was not non exceptional. And exceptional is already a negation, it's something not normal. It's not not not normal.

Reply:
I will change to `exceptional`

Reply:
or `critical`

Reply:
hasCriticalError ?